docs/source/userdoc/requirements/interns/adna-njuhovic/GPIO_requirements/functional_requirements.rst

```diff
@@ -0,0 +1,55 @@
+===============================
+Functional Requirements
+===============================
+
+FR-01: Pin Configuration
```

Comment:
ovo je potrebno napisati kao jednostavnu recenicu, ne sa tezama i razdvojiti na vise rqs; 
npr. ovdje mozes jedan rq samo ovaj dio "Each GPIO pin must be configurable to one of four modes: input, output, analog and alternate function", drugi bi bio definicija za sta se koristi MODER registar
takodjer, ovo bi bili informacijski, a ne funkcionalni rqs